config/environments/development.rb

```diff
@@ -52,6 +52,8 @@
   # number of complex assets.
   config.assets.debug = true
   config.assets.compile = true
+  
+  config.web_console.whitelisted_ips = '69.204.8.105'
```

Comment:
oof should this go in config?

Reply:
This was necessary for me and it doesn't hurt? I guess I can omit this as well.

Reply:
im not sure if it's sens- i t-ve data or not